### PR TITLE
Update docs for v0.12.0 release

### PR DESCRIPTION
**Pull request by zhongyi-zhang**

### CODE REVIEW

**Comment by zhongyi-zhang** [comment]
@bingosummer @norshtein could you help review?

File: docs-content/index.html.md.erb [hunk 1]
```diff
@@ -21,16 +21,16 @@ See which Azure services are supported below.
...
 * [Azure Key Vault](https://azure.microsoft.com/en-us/services/key-vault/)
 * [Azure Redis Cache](https://azure.microsoft.com/en-us/services/cache/)
 * [Azure Service Bus](https://azure.microsoft.com/en-us/services/service-bus/)
 * [Azure Storage](https://azure.microsoft.com/en-us/services/storage/)
+* [Azure Text Analytics (Cognitive Services)](https://azure.microsoft.com/en-us/services/cognitive-services/text-analytics/)
 
 Service modules with `Preview` (Beta) stability, which is intended for testing during proof of concept stage:
 
-* None
+* [Azure CosmosDB](https://azure.microsoft.com/en-us/services/cosmos-db/)
```
Comment (norshtein) [inline docs-content/index.html.md.erb:33]: Redis cache is also in preview now

Comment (norshtein) [reply docs-content/index.html.md.erb:33]: OK... seems in OSBA v1.2.0, Redis cache is not made preview,

Comment (zhongyi-zhang) [reply docs-content/index.html.md.erb:33]: Yeah, Redis got preview after cutting v1.2.0.

**Comment by norshtein** [comment]
Should the usage of MSSQL server from registered also be added to the doc? I think it is an important feature.

**Comment by zhongyi-zhang** [comment]
The using doc for the tile has only one page. So I think we only put an example for the simplest usage here. For any advanced features, users should jump to GitHub from the link in each service module.

**Comment by norshtein** [comment]
OK, it makes sense.

**Review by norshtein — APPROVED** [review]
LGTM

**Review by bingosummer — APPROVED** [review]

**Comment by jknostman3** [comment]
@zhongyi-zhang Thanks!

File: docs-content/index.html.md.erb (hunk 1 above)
Comment (jncd) [reply docs-content/index.html.md.erb:33]: Hi @zhongyi-zhang 
Can you check the name of your product?
On the Azure site (https://azure.microsoft.com/en-us/services/cosmos-db/), I see it spelled "Azure Cosmos DB" but in these docs you've written "Azure CosmosDB". 
Probably, you want to check and correct this — gotta respect your product name.

Comment (zhongyi-zhang) [reply docs-content/index.html.md.erb:33]: @jncd thanks for raising it. I'll create a PR to fix it soon.

Comment (zhongyi-zhang) [reply docs-content/index.html.md.erb:33]: https://github.com/pivotal-cf/docs-azure-open-service-broker-pcf/pull/6

**Pull request merged**